README.rst

```diff
@@ -1,55 +1,49 @@
 |Build Status| |Codecov| |license|
 
-frontend-template-application
+frontend-app-learner-record
 =================================
 
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+Please tag **@edx/aperture-eng** on any PRs or issues.  Thanks.
 
 Introduction
 ------------
 
-This repository is a template for Open edX micro-frontend applications. It is flagged as a Template Repository, meaning it can be used as a basis for new GitHub repositories by clicking the green "Use this template" button above.  The rest of this document describes how to work with your new micro-frontend after you've created a new repository from the template.
+This is the Learner Record micro-frontend, currently under development by `edX <https://www.edx.org>`_.
 
-After Copying The Template
---------------------------
+The Learner Record provides information about the enrolled programs for a user.
 
-You'll want to do a find-and-replace to replace all instances of ``frontend-template-application`` with the name of your new repository.  Also edit index.html to replace "Application Template" with a friendly name for this application that users will see in their browser tab.
-
-**Prerequisite**
-
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
-
-**Installation and Startup**
-
-In the following steps, replace "frontend-template-application' with the name of the repo you created when copying this template above.
-
-1. Clone your new repo:
-
-  ``git clone https://github.com/edx/frontend-template-application.git``
+Installation
+------------
 
-2. Use node v12.x.
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
```

Comment:
I'm not sure if this statement is true? I don't think this is currently bundled with devstack?

I think we would provide instructions to pull, build, and start the Learner Record MFE here.

Reply:
I don't think it's true either. I updated the instructions so it's more focused on the MFE.